### PR TITLE
[A7] style: improve pep8 linting

### PR DESCRIPTION
**Pull request by pedrokiefer**

### CODE REVIEW

File: owasp-top10-2017-apps/a7/gossip-world/app/model/db.py [hunk 1]
```diff
@@ -2,202 +2,229 @@
...
+
+
 class DataBase:
-    def __init__ (self, host, user, password, database):
+    def __init__(self, host, user, password, database):
         self.host = host
         self.user = user
         self.password = password
         self.database = database
-        self.db = MySQLdb.connect(host=self.host,user=self.user,passwd=self.password,db=self.database,charset='utf8')
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a7/gossip-world/app/model/db.py]: Did you miss this following line or you consider it as an unnecessary one? 😃
```python
        self.db = MySQLdb.connect(host=self.host,
                                  user=self.user,
                                  passwd=self.password,
                                  db=self.database,
                                  charset='utf8')
```

Comment (pedrokiefer) [reply owasp-top10-2017-apps/a7/gossip-world/app/model/db.py]: I've changed it to a call to `self.connect()` which does the same thing.

We could refactor it further, ensuring that the creator of `DataBase` calls `db.connect()`, this way we are not trying to connect on the constructor method of the class.

**Comment by vitoriario** [comment]
Thank you @pedrokiefer, the code looks much better!

File: owasp-top10-2017-apps/a7/gossip-world/app/model/db.py (hunk 1 above)
Comment (rafaveira3) [reply owasp-top10-2017-apps/a7/gossip-world/app/model/db.py]: That's awesome! Thanks for this info. 😃

**Pull request merged**